### DISCUSSION TITLE
Open Live Preview by default

### DISCUSSION THREAD

**Discussion by JorisOrangeStudio** [statamic/cms#8662] [Q&A] [answered]
Is it possible to open the Live Preview of an entry by default when opening the entry in the CP? Is it maybe possible to trigger that action (to open Live Preview) via javascript?

**Comment by jasonvarga** [comment, marked as answer]
Nothing native to do it, but yes you could use some Javascript. This seems to work.

```js
setTimeout(() => document.querySelector('.publish-tab-actions button').click(), 1000);
```
